apps/meteor/server/models/raw/Users.js

```diff
@@ -3079,4 +3079,17 @@ export class UsersRaw extends BaseRaw {
 			},
 		);
 	}
+
+	countActiveUsersWithCustomRoles() {
+		const customRoleRegex = /^[0-9a-fA-F]{24}$/;
+		const query = {
+			active: true,
+			roles: {
```

Comment:
Can you post an `explain` of this one? Just to check if the query can use a suitable index for this kind of filter, cause `active` is not a very discriminative filter.

Reply:
Just posted